Bug fixes for package controllers

chrden opened the pull request:
Added close function for positional content settings editor
Added null check for positional content controller
Removed null '.items' from breakpoint controller

chrden committed:
Bug fixes for package controllers

Added close function for positional content settings editor
Added null check for positional content controller
Removed null '.items' from breakpoint controller

hifi-phil commented on PositionalContent.Code/Web/UI/App_Plugins/PositionalContent/contentEditor/positionalcontenteditor.controller.js:
```diff
@@ -163,7 +163,13 @@ angular.module("umbraco").controller("HiFi.PositionalContentDialogController", [
 
         };
 
```
I know you're being safe here but because v8 changes to editorService I think  it can just be

 $scope.close = function () {
     editorService.close();
 }

hifi-phil commented on PositionalContent.Code/Web/UI/App_Plugins/PositionalContent/positionalcontent.controller.js:
```diff
@@ -97,8 +97,10 @@ angular.module("umbraco").controller("HiFi.PositionalContent.Controller", [
             .then(function (dataType2) {
-                if (dataType2.guid)
-                    $scope.model.value.dtdGuid = dataType2.guid;
+                if (dataType2) {
```
can we simplify to just 

if(dataType2 && dataType2.guid)  ??

hifi-phil reviewed (changes requested):
Looks great thanks, I've left a few comments below

chrden committed:
Amends as suggested by Phil

Removed if else on close function of editor controller
Removed second unnecessary if and combined into one on main controller
